internal/controller/imagerepository_controller.go

```diff
@@ -442,7 +461,7 @@ func (r *ImageRepositoryReconciler) ProvisionImageRepository(ctx context.Context
 	imageRepository.Status.Image.URL = quayImageURL
 
 	if imageRepository.Spec.Image.Visibility == "" {
-		imageRepository.Spec.Image.Visibility = imagerepositoryv1alpha1.ImageVisibilityPublic
+		imageRepository.Spec.Image.Visibility = imagerepositoryv1alpha1.ImageVisibilityPrivate
```

Comment:
I'm not sure about the change.
Having all images private by default requires a paid plan on quay, which makes defaults not usable in upstream Konflux. So admins would have to pay for a quay plan or specify public visibility for every image, which is not a user friendly approach.

I suggest to have it configurable via an env var. If the env var is not set, it means public (to not to break current behavior).